### ISSUE TITLE
Client side - URI Issue - Not getting whole URI into  the request

### ISSUE THREAD

**Issue by ejChathuranga** [crossbario/autobahn-java#425]
My WSS URI and PORT: wss://xxxxxx-xxxxxx.maindomain.com/websocket:9050

When I'm using `wss://echo.websocket.org` URI, It's sending the request to the server and return only 200. When I debugged I found, `InetAddress` class have been used in `(new InetSocketAddress(mWsHost, mWsPort)).getAddress().getHostName())` and will return HOST name as `echo.websocket.org`. 

My server is configured WS for `xxxxxx-xxxxxx.maindomain.com/websocket` URI. So when parsing HOST name as above URI into `(new InetSocketAddress(mWsHost, mWsPort)).getAddress().getHostName())`  method, It will not return IP for the mentioned URI. Does there any given definitions for naming URI for the web sockets?. Or is there any solutions to pass this through

**Comment by oberstet** [comment]
just provide the complete URL to connect, as `wsUri` is parsed into `mWsPath`

https://github.com/crossbario/autobahn-java/blob/master/autobahn/src/main/java/io/crossbar/autobahn/websocket/WebSocketConnection.java#L326

and that is used in the websocket opening handshake as well:

https://github.com/crossbario/autobahn-java/blob/master/autobahn/src/main/java/io/crossbar/autobahn/websocket/WebSocketConnection.java#L162

**Comment by ejChathuranga** [comment]
I used my complete 
URL: wss://xxxxxx-xxxxxx.maindomain.com/websocket 
Port: 9050

Then it's take `mWsUri = wss://xxxxxx-xxxxxx.maindomain.com/websocket:9050`
and `mWsPath =  /websocket:9050`

Then it's return those debugs. 
`D/NetworkSecurityConfig: No Network Security Config specified, using platform default`
`D/io.crossbar.autobahn.websocket.WebSocketConnection: fail connection [code = 2, reason = failed to connect to xxxxxx-xxxxxx.maindomain.com/##.###.##.### (port 9050) from /192.168.8.111 (port 38171) after 6000ms`
   ` mReader already NULL`
    `mWriter already NULL`
`D/io.crossbar.autobahn.websocket.WebSocketConnection: Socket already closed`
    `mReader already NULL`
`D/io.crossbar.autobahn.websocket.WebSocketConnection: Worker threads stopped`

As I debugged this issue arises when trying to connect to the socket. It's catching on the try catch and throw and return. Any suggestions?
